index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div class="container">
```

Comment:
It would be better if you use semantic HTML elements because it makes your code easier to read and help the search engine.

Reply:
[OPTIONAL] Great job on this one 👍🏽 explaining the why of the changes you're requesting. To make this even better you could also provide some links to documentation explaining in more detail. 🤓